api/app/signals/apps/api/tests/test_private_signal_endpoint_create.py

```diff
@@ -331,6 +331,7 @@ def test_create_initial_child_signals_copy_attachments(self, validate_address):
         for note, attachment in zip(note_qs, attachment_qs):
             filename = os.path.basename(attachment.file.name)
             self.assertEqual(f'Foto gekopieerd van hoofdmelding: {filename}', note.text)
```

Comment:
"Foto" or "bijlage"? In the serializer you are adding a note with "Bijlage" instead of "Foto" so perhaps this was an oversight. If so I would have expected this test to fail 🤔 however it passed 🤔 

However I now see that this is for the copy's made for attachments when making "deelmedlingen". Shouldn't this also be renamed to "Bijlage"?

Reply:
It should be "bijlage" throughout - I'll check why this test passes when it should not.

Reply:
Both message and test were not updated for the "copy-from-parent" flow. They are now. The only spot that still mentions photos is in the PDFs - there I will leave it for now because the PDF rendering has no support for anything other that photo's (or more precisely images).